eras/alonzo/test-suite/cardano-ledger-alonzo-test.cabal

```
@@ -54,7 +54,7 @@ library
         cardano-ledger-shelley-test ^>=1.1,
         cardano-ledger-mary ^>=1.1,
         cardano-ledger-shelley-ma-test ^>=1.1,
-        cardano-protocol-tpraos ^>=1.0,
+        cardano-protocol-tpraos ^>=1.1,
```

You see, this bump is not needed, because there is no direct dependency on the `testlib`, which is what changed, right?  ``cardano-protocol-tpraos` library have not changed.
```suggestion
        cardano-protocol-tpraos ^>=1.0,
```